scripts/freshness-guard.sh

```
@@ -61,16 +63,24 @@ if [ "$MODE" = "--fix" ]; then
   if [ -x "$PY" ]; then
     ( cd "$THEME" && "$PY" data/gen-design-tokens.py && "$PY" data/build-collection-sot.py \
         && "$PY" data/gen-collection-hub.py ) >/tmp/fg_fix_sot.log 2>&1 \
-      && c_ok "regenerated SOT (design-tokens + sot.json + hubs)" \
+      && c_ok "regenerated collection SOTs (design-tokens + collection sot.json + hubs)" \
       || { c_bad "SOT regeneration failed (see /tmp/fg_fix_sot.log)"; tail -8 /tmp/fg_fix_sot.log | sed 's/^/    /'; }
+    ( cd "$ROOT" && "$PY" scripts/build-lookbook-sot.py ) >/tmp/fg_fix_lookbook_sot.log 2>&1 \
+      && c_ok "regenerated lookbook-sot.json" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)